impl/src/main/java/dao/Vote.java

```diff
@@ -0,0 +1,48 @@
+package dao;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+
+public class Vote {
+
+    long userId;
```

Comment:
a lot of spacing here :)

Reply:
Not more than anywhere else, as far as I can see. :P